docs/design/database.md

```diff
@@ -45,24 +44,38 @@ TransactionHash {
     H256 TxHash "PK"
     u64 TxNumber
 }
-TxChangeIdIndex {
+TxTransitionIdIndex {
     u64 TxNumber "PK"
-    u64 ChangeId 
+    u64 TransitionId 
 }
-BlockChangeIdIndex {
+BlockTransitionIdIndex {
     u64 BlockNumber "PK"
-    u64 ChangeId
-}
-ChangeSet {
-    u64 ChangeId "PK"
-    ChangeSet PreviousValues "[Acc1[Balance,Nonce),Acc2(Balance,Nonce)] Previous values"
+    u64 TransitionId
 }
 History {
     H256 Account "PK"
-    u64 ChangeIdList "[ChangeId,ChangeId,...] Points where account changed"
+    u64 TransitionIdList "TransitionId[] (transitions where account changed)"
+}
```

Comment:
Same as we have two `ChangeSet` tables we have two `History` tables `AccountHistory` and `StorageHistory`

```
AccountHistory {
    H256 Account "PK"
    u64 TransitionIdList "TransitionId[] (transitions where account changed)"
}
StorageHistory {
    H256 Account "PK"
    H256 StorageKey "PK"
    u64 TransitionIdList "TransitionId[] (transitions where account changed)"
}
```

Reply:
good catch, fixing